Sources/VLCUI/VLCVideoPlayer/Proxy.swift

```
@@ -152,5 +152,42 @@ public extension VLCVideoPlayer {
         public func playNewMedia(_ newConfiguration: Configuration) {
             videoPlayerView?.setupVLCMediaPlayer(with: newConfiguration)
         }
+
+        /// Starts the recording process
+        /// - Parameter directoryPath: The directory path where the recording will be saved
+        /// - Returns: Indicates whether the recording started successfully
+        public func startRecording(directoryPath: String) -> Bool {
```

```suggestion
        public func startRecording(atPath path: String) -> Bool {
```

---

Instead of it returning a `Bool`, I would rather we throw an error. We will have to define `VLCVideoPlayer.Error`.

---

I preferred this approach because the VLC player returns a Bool, but even if the start recording is successful, it returns false. So, instead of directly using the returned value, I had to provide my own. I’m not entirely sure about the reason.